migration/1646303882607-seedTokes.ts

```diff
@@ -0,0 +1,160 @@
+         ('Amp','AMP','0xff20817765cb7f73d4bde2e66e067e58d11095c2',18, 1),
+         ('Ankr Network','ANKR','0x8290333cef9e6d528dd5618fb97a76f268f3edd4',18, 1),
+         ('Axie Infinity Shard','AXS','0xbb0e17ef65f82ab018d8edd776e8dd940327b28b',18, 1),
+         ('BarnBridge Governance Token','BOND','0x0391D2021f89DC339F60Fff84546EA23E337750f',18, 1),
+         ('Cryptex','CTX','0x321c2fe4446c7c963dc41dd58879af648838f98d',18, 1),
+         ('Fantom Token','FTM','0x4e15361fd6b4bb609fa63c81a2be19d873717870',18, 1),
+         ('Graph Token','GRT','0xc944e90c64b2c07662a292be6244bdf05cda44a7',18, 1),
+         ('Injective Token','INJ','0xe28b3B32B6c345A34Ff64674606124Dd5Aceca30',18, 1),
+         ('Livepeer Token','LPT','0x58b6a8a3302369daec383334672404ee733ab239',18, 1),
+         ('Moss Carbon Credit','MCO2','0xfc98e825a2264d890f9a1e68ed50e1526abccacd',18, 1),
+         ('Orchid','OXT','0x4575f41308EC1483f3d399aa9a2826d74Da13Deb',18, 1),
+         ('The Sandbox','SAND','0x3845badAde8e6dFF049820680d1F14bD3903a5d0',18, 1),
+         ('Skale','SKALE','0x00c83aecc790e8a4453e5dd3b0b4b3680501a7a7',18, 1),
+         ('Smooth Love Potion','SLP','0xcc8fa225d80b9c7d42f96e9570156c65d6caaa25',18, 1),
+         ('Somnium Space Cubes','CUBE','0xdf801468a808a32656d2ed2d2d80b72a129739f4',18, 1),
+         ('Storj','STORJ','0xb64ef51c888972c908cfacf59b47c1afbc0ab8ac',18, 1),
+         ('Wrapped LUNA Token','LUNA','0xd2877702675e6ceb975b4a1dff9fb7baf4c91ea9',18, 1),
+         ('UMA Voting Token v1','UMA','0x04Fa0d235C4abf4BcF4787aF4CF447DE572eF828',18, 1),
+         ('Uniswap','UNI','0x1f9840a85d5af5bf1d1762f925bdaddc4201f984',18, 1),
+         ('Wrapped Centrifuge','wCFG','0xc221b7e65ffc80de234bbb6667abdd46593d34f0',18, 1),
+         ('Audius','AUDIO','0x18aAA7115705e8be94bfFEBDE57Af9BFc265B998',18, 1),
+         ('Mask Network','MASK','0x69af81e73a73b40adf4f3d4223cd9b1ece623074',18, 1),
+         ('Radicle','RAD','0x31c8eacbffdd875c74b94b077895bd78cf1e64a3',18, 1),
+         ('API3','API3','0x0b38210ea11411557c13457D4dA7dC6ea731B88a',18, 1),
+         ('The Burn Token','ASH','0x71590d4ed14d9cbacb2cff8abf919ac4d22c5b7b',18, 1),
+         ('SuperRare','RARE','0xba5bde662c17e2adff1075610382b9b691296350',18, 1),
+         ('Fetch','FET','0xaea46A60368A7bD060eec7DF8CBa43b7EF41Ad85',18, 1),
+
+        
+         ('Ethereum native token','ETH','0x0000000000000000000000000000000000000000',18, 3),
+         ('DAI Ropsten','DAI','0xad6d458402f60fd3bd25163575031acdce07538d',18, 3),
+         ('Giveth Ropsten Test','YAY','0x067eA48882E6D728A37acfd1535ec03f8E33794a',18, 3),
+         ('UNI Ropsten','UNI','0x1f9840a85d5af5bf1d1762f925bdaddc4201f984',18, 3),
+         
+         
+         ('Xdai native token','XDAI','0x0000000000000000000000000000000000000000',18, 100),
+         ('Giveth Token','GIV','0x4f4F9b8D5B4d0Dc10506e5551B0513B61fD59e75',18, 100),
+         ('Curve DAO Token','CRV','0x1337BedC9D22ecbe766dF105c9623922A27963EC',18, 100),
```

Comment:
We should revise our approach in inserting tokens. Consider this case, the address has upper case and lower case characters. But in finding tokens we have assumed all characters are lowercase.